src/dymoprint/lib/utils.py

```diff
@@ -33,3 +31,18 @@ def draw_image(bitmap):
         yield drawobj
     finally:
         del drawobj
+
+
+def px_to_mm(px):
+    mm = px / PIXELS_PER_MM
+    # Round up to nearest 0.1mm
+    return math.ceil(mm * 10) / 10
+
+
+@contextlib.contextmanager
+def system_run():
+    try:
+        yield
+    except Exception as e:  # noqa: BLE001
+        print_exception(e)
```

Comment:
Thanks for adding the `--verbose`!

What do you think about adding something like this?

```suggestion
        print_exception(e)
        if not is_verbose():
            LOG.error("Run with --verbose for more information")
```

Also, for the environment variables, as far as I can tell `DEBUG` and `VERBOSE` are equivalent, and I think there's potential for confusion to have both. How would you feel about removing `DEBUG` and just keeping `VERBOSE`?

Reply:
Good idea. Done.